functions/supplier/pom.xml

```diff
@@ -27,6 +27,7 @@
         <module>s3-supplier</module>
         <module>twitter-supplier</module>
         <module>cdc-debezium-supplier</module>
+        <module>cdc-debezium-supplier2</module>
```

Comment:
I don't think we have to do this. 
We are in a new major `4.0` version, so it is fully OK to remove the old code and functionality and apply a new one.
Or you did it deliberately to simplify a review and we will have a follow up PR to clean things up?

Reply:
If the consensus is to remove the old code (i'm strongly for it) I'll remove the `cdc-debezium-common` , `cdc-debezium-boot-starter`, `cdc-debezium-supplier` and `cdc-debezium-source`  projects and 
will make the new `cdc-debezium-supplie2` and `cdc-debezium-source2`  into `cdc-debezium-supplier` and `cdc-debezium-source` instead.

Reply:
resolved